src/api/Feed.js

```diff
@@ -1265,6 +1366,55 @@ class Feed extends Base {
         return null;
     };
 
+    /**
+     * Replace a reply of feed item with new comment data.
+     *
+     * @param {Object} replyUpdates - New data to be applied to the reply.
+     * @param {string} parentId - ID of the parent feed item.
+     * @param {string} id - ID of the reply to replace.
+     * @return {void}
+     */
+    updateReplyItem = (replyUpdates: Object, parentId: string, id: string) => {
```

Comment:
Should this also set `isPending` on the reply when updating the reply message?

Reply:
This sets whatever is passed in `replyUpdates` param. It works in similar way as `updateFeedItem`. In future PR I will create a handler for updating replies that will utilize `updateReplyItem` and will also pass `isPending: true`.